### ISSUE TITLE
Missing link to Understanding client side JavaScript frameworks

### ISSUE THREAD

**Issue by CloudZero458** [mdn/content#26590]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks

### What specific section or headline is this issue about?

The "Understanding client-side JavaScript frameworks" page

### What information was incorrect, unhelpful, or incomplete?

The sidebar (on the left) is missing a direct link to the **page** "Understanding client-side JavaScript frameworks" , which is located in the "Introduction to client-side frameworks" **section**. It's actually the first page in the section (the overview page), but the only way to access it is to go the **page** "Introduction to client-side frameworks", and then click the "Overview: Client-side JavaScript frameworks" link at the top. To clarify there is both a **section** called "Introduction to client-side frameworks" and a **page** called "Introduction to client-side frameworks" under that section

### What did you expect to see?

I would like to see  a link to the "Understanding client-side JavaScript frameworks"overview page in the sidebar

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/tools_and_testing/client-side_javascript_frameworks`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/index.md
* Last commit: https://github.com/mdn/content/commit/4def230f85756724b59660e3cd9de363db724ef8
* Document last modified: 2023-02-24T00:05:28.000Z

</details>

**Comment by hamishwillee** [comment]
Thanks for reporting. Based on the name, I suspect [Understanding client-side JavaScript frameworks](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks)  is supposed to live above [Introduction to client-side frameworks](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Introduction)

![image](https://user-images.githubusercontent.com/5368500/236353599-e48e0b89-99d3-495d-b9b2-37d7934a00f4.png)

But there is a huge amount of duplication between those docs, which is a maintenance cost. @wbamberg @Josh-Cena Any point in keeping [Understanding client-side JavaScript frameworks](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks)? The only bit I see of any "additional" value, possibly, is the introductory section.

**Comment by CloudZero458** [comment]
> I started a discussion about this corner of Learn a bit ago, but noone seemed to care much: https://github.com/orgs/mdn/discussions/310.

I think that suggestion is a better way to organize things. I do like the tutorials for the frameworks, both client and server side and I hope  they stay on MDN

**Comment by hamishwillee** [comment]
@wbamberg I've added a comment in support. But I still want to kill or seriously cut back one of these docs, because the duplication is extreme. I guess if we reorg it could fall out of that.

**Comment by wbamberg** [comment]
Yeah, I think we could 

> @wbamberg I've added a comment in support. But I still want to kill or seriously cut back one of these docs, because the duplication is extreme. I guess if we reorg it could fall out of that.

It's a while since I made this proposal so I don't remember all the details, but IIRC the proposal would help with this duplication, because at the moment there isn't a top-level "React" page, but in my proposal at https://github.com/orgs/mdn/discussions/310 we create these top-level pages. And the content for these pages could come out of e.g. https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks#react_tutorials. And once you'd done that for all the frameworks, there isn't much left of https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks.

The [model I tried to come up with for Learn when I redid the sidebar](https://github.com/mdn/yari/blob/main/kumascript/macros/LearnSidebar.ejs#L340-L806) was a kind of three-level structure:
- top level is areas like "JavaScript"
- inside there are modules like ["JS Objects"](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects)
- modules consist of individual lesson pages

This works all right for most of Learn, but breaks down a bit for client-side frameworks (hence https://github.com/orgs/mdn/discussions/310), and also for "Common questions", but that's blocked by Mozilla : https://github.com/mdn/yari/issues/8365 🤷 .

**Comment by postmeback** [comment]
Any decision taken ?

**Comment by Josh-Cena** [comment]
The learning area has been refactored and this issue is no longer relevant.
